src/Output/Copy/Serializer/QuerySerializer.php

```diff
@@ -15,6 +16,8 @@ public function serializeRequest(RequestInterface $request): string
         foreach ($request->getRawQueryParameters() as $name => $value) {
             if ($value === null) {
                 continue;
+            } elseif ($value instanceof DateTimeInterface) {
+                $value = $value->format(DateTimeInterface::RFC3339);
```

Comment:
Any particular reason to not use the ATOM const which is a more developer-friendly name for the same format?

Reply:
Yes, they are the same but OpenAPI documentation uses this RFC number:
https://swagger.io/specification/ check 'Data-types' (sorry, this documentation is a shame)